test/metrics.test.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+var assert = require('proclaim');
+var metrics = require('../lib').constructor.metrics;
+var sinon = require('sinon');
+
+describe('metrics', function() {
+  var xhr;
+  var spy;
+
+  beforeEach(function() {
+    xhr = sinon.useFakeXMLHttpRequest();
```

Comment:
is the reason the tests are failing that this isn't IE9 compatible?

https://caniuse.com/#feat=xhr2

Reply:
it could be! i'll look into it a bit more. Unfortunately, the IE tests have been flaky even before this change though